### ISSUE TITLE
MQ without MQ for z/OS

### ISSUE THREAD

**Issue by danielbsan** [cicsdev/cics-java-liberty-mq-jms#11]
Hi guys
Can I use your client example MQ client application within CICS Liberty and connect to an MQ Server outside of z/OS without having MQ for z/OS installed? Just with Resource Adapter (rar)?
Thank you

**Comment by PhilWakelin** [comment]
Yes this is supported and the relevant MQ RAR can be downloaded from Fix Central as documented here https://www.ibm.com/docs/en/ibm-mq/9.3?topic=adapter-installing-mq-resource
The instructions for installing the MQ RAR are here https://www.ibm.com/docs/en/ibm-mq/9.3?topic=adapter-installing-mq-resource

**Comment by PhilWakelin** [comment]
Closing as resolved
